base/commands/job/job_submit.go

```diff
@@ -1,36 +1,26 @@
 package job
 
 import (
-	"bytes"
 	"context"
-	"crypto/sha256"
 	"fmt"
-	"io"
 	"math"
-	"math/rand"
-	"os"
 	"path/filepath"
 	"strings"
 	"time"
 
 	"github.com/hazelcast/hazelcast-go-client"
-	"github.com/hazelcast/hazelcast-go-client/cluster"
-	"github.com/hazelcast/hazelcast-go-client/types"
 
 	"github.com/hazelcast/hazelcast-commandline-client/clc"
 	"github.com/hazelcast/hazelcast-commandline-client/clc/cmd"
 	"github.com/hazelcast/hazelcast-commandline-client/clc/paths"
 	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/jet"
 	"github.com/hazelcast/hazelcast-commandline-client/internal/log"
 	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
-	"github.com/hazelcast/hazelcast-commandline-client/internal/proto/codec"
 )
 
 const (
-	// see: https://github.com/hazelcast/hazelcast/issues/24285
-	envExperimentalCalculateHashWorkaround = "CLC_EXPERIMENTAL_WORKAROUND_24285"
-	minServerVersion                       = "5.3.0-BETA-2"
-	defaultBatchSize                       = 2 * 1024 * 1024 // 10MB
+	minServerVersion = "5.3.0"
```

Comment:
it is not directly related to this line, but maybe you would like to use `github.com/hashicorp/go-version` package to compare versions instead of `internal/version.go`.